README.md

```diff
@@ -128,17 +148,222 @@ To start Parity Ethereum as a regular user using `systemd` init:
 2. Copy release to bin folder, write `sudo install ./target/release/parity /usr/bin/parity`
 3. To configure Parity Ethereum, write a `/etc/parity/config.toml` config file, see [Configuring Parity Ethereum](https://paritytech.github.io/wiki/Configuring-Parity) for details.
 
-## Parity Ethereum toolchain
+## 4. Documentation <a id="chapter-004"></a>
+
+Official website: https://parity.io
+
+Be sure to [check out our wiki](https://wiki.parity.io) for more information.
+
+### Viewing documentation for Parity Ethereum packages
+
+You can generate documentation for a Parity Ethereum Rust package and have it automatically open in your web browser using [rustdoc with Cargo](https://doc.rust-lang.org/rustdoc/what-is-rustdoc.html#using-rustdoc-with-cargo), (of the The Rustdoc Book), by running the the following commands:
+
+```
+cargo doc --package <spec> --open
+```
+
+Replacing `<spec>` with one of the following (i.e. `cargo doc --package parity-ethereum --open`):
```

Comment:
I don't think this section is that useful, would rather see it in some top-level code organisation overview section.

Reply:
I think it's handy if you just want to copy/paste the relevant docs that you want to open in development (without opening them all), but it does take up a lot of space. I've wrapped it in a "Details >" section 

I've made changes in commit 55d1a3e2cf677a4c37a84d19bf3e6972ff5c6d57

What are you envisioning by a top-level code organisation overview section?

I think ideally there'd be a Parity Ethereum equivalent of https://docs.substrate.dev/docs/architecture-of-a-runtime, where you could just click on one of the packages and it'd take you to where it's published on crates.io or similar.

And that there'd be an equivalent of say https://docs.substrate.dev/docs/srml-overview#section-srml-architecture that contains your architecture diagram.

Reply:
> I think ideally there'd be a Parity Ethereum equivalent of https://docs.substrate.dev/docs/architecture-of-a-runtime, 

That looks really awesome.